### ISSUE TITLE
Crashes with sound channel increase.

### ISSUE THREAD

**Issue by xsoalokinx** [brndd/vanilla-tweaks#22]
Getting pretty frequent crashes when using anything greater than 16 for sound channels.
Client is 1.16.5 (7060)
Platform is Windows 10. 
Release used is v1.6.0.
Also using https://github.com/hannesmann/vanillafixes v1.3 (DXVK).
Have also heard of others crashing when using vanilla-tweaks.
Perhaps setting the default sound channels to 64 is not a good idea?

**Comment by brndd** [comment]
Judging by the client version, you're playing on Turtle WoW, am I correct? Perhaps this is a new issue prompted by some new TWoW update, as >16 sound channels definitely used to be stable in the past.

**Comment by cpevors** [comment]
Can confirm that, on a non-Turtle WoW client, >16 is still stable.

**Comment by xsoalokinx** [comment]
> Owner



> Judging by the client version, you're playing on Turtle WoW, am I correct? Perhaps this is a new issue prompted by some new TWoW update, as >16 sound channels definitely used to be stable in the past.

Yep, I am playing on turtle wow. Also I can't say for certain that the crashes are caused by increasing sound channels, but it definitely appears to be the case. I have tested with 32,64. And crashes can happen quite sporadically and consistently. I will do some testing without the dxvk version of vanillafixes.

**Comment by welpenhund** [comment]
I recently had also random crashes/freezes (playing on t-wow) and i wondered already why and how because it never happened before. Now i seen this threat here, will also try using only 16 channels for sound and report if i still have this freezes/crashes.
Was spending hours testing my hardware and scanned my memory cause i thought it's that, but it only crashed in WoW...

**Comment by welpenhund** [comment]
24 hours tested - no crashes anymore after reducing the sound channels to 16.... thanks at the post creator cause i thought its my hardware that is screwed.
